suite-native/atoms/src/SwipeableWalkthrough/SwipeableWalkthroughStep.tsx

```diff
@@ -83,15 +83,15 @@ export const SwipeableWalkthroughStep = ({
                 walkthroughStepAnimatedStyle,
                 applyStyle(stepContainerStyle, { height: stepContainerHeight }),
             ]}
-            testID="@deviceOnboarding/SwipeableWalkthroughStep"
+            testID="swipeableWalkthroughStep"
```

Comment:
nitpick, just for consistency
```suggestion
            testID="@swipeableWalkthroughStep"
```

Reply:
https://github.com/trezor/trezor-suite/pull/19963/commits/1b1a1219aca94a82c10fa5da8185455069f85627